config/env.sample

```diff
@@ -0,0 +1,5 @@
+PORT=5005
+CERT_FILE="config/public-cert.pem"
+KEY_FILE="config/private-key.pem"
+KEY_PASS=KEY_FILE
```

Comment:
`KEY_PASS` is for if your SSL certificate's private key requires a password for usage. There's no way to default it, and probably 75% chance the user won't bother to set one anyway, so it should simply be left out of any sample config.